clients/web/src/app.d.ts

```diff
@@ -3,24 +3,32 @@
 
 // See https://kit.svelte.dev/docs/types#app
 // for information about these interfaces
-import type {SessionUser} from "$lib/types/SessionUser";
+
+import type {SSRData} from "@urql/core/dist/types/exchanges/ssr";
```

Comment:
Do we use this anywhere? If it is internal to URQL, we probably don't need it in our declarations

Reply:
Its used here, I added the type to get rid of an error